scripts/travis_yml_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test to ensure travis.yml file and protractor.conf.js have the same test suites"""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+from core.tests import test_utils
+import python_utils
+
+
+class TravisCIFileTests(test_utils.GenericTestBase):
```

Comment:
Firstly write a new method which will check the travis yaml is in expected form and then write the test for the new method.

Reply:
Hey @DubeySandeep I don't understand what is the expected form. It's in yaml already, what should i check for.